core/src/main/scala/chisel3/experimental/hierarchy/Instance.scala

```diff
@@ -16,7 +16,7 @@ import firrtl.annotations.IsModule
   *
   * @param underlying The internal representation of the instance, which may be either be directly the object, or a clone of an object
   */
-final case class Instance[+A] private [chisel3] (private[chisel3] underlying: Underlying[A]) extends SealedHierarchy[A] {
+final case class Instance[+A] /*private [chisel3]*/ (private[chisel3] underlying: Underlying[A]) extends SealedHierarchy[A] {
```

Comment:
I believe this is unnecessary because of the availability of `LibraryHooks.buildInstance`, that's a much more obnoxious API which is intentional because we don't want normal users doing this (at least, at the moment).

Reply:
yep pulling that in